.github/workflows/alert_terraform_drift.yml

```diff
@@ -0,0 +1,54 @@
+name: Alert Terraform Changes
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 12 * * *"
```

Comment:
Would you please change this to run sometime other than on the hour?

Reply:
21 12 * * *